### DISCUSSION TITLE
Welcome to undici Discussions!

### DISCUSSION THREAD

**Discussion by mcollina** [nodejs/undici#655] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ThakurKarthik** [comment]
Hello everyone, I am Karthik, I work on a nodejs + vue powered application at my workplace. I am very interested to learn more about the "how http works" and recently I am going through http specification and some other links. Since Undici is a http client I am also peeking into its internals and looking forward to listen around discussions of the evolution of this project in the upcoming days

Regards,
Karthik
